ui/src/reducers/__tests__/index.test.js

```diff
@@ -135,7 +135,7 @@ describe('reducer', () => {
   ];
 
   it('should return the initial state', () => {
-    expect(reducer(undefined, {})).toEqual(initialState);
+    expect(JSON.stringify(reducer(undefined, {}))).toEqual(JSON.stringify(initialState));
```

Comment:
Any reason for the `JSON.stringify` call, or is it just for inspection when testing PR?

Reply:
The test breaks otherwise. It's a jest issue: https://github.com/facebook/jest/issues/5998 That's the workaround they suggest.

Reply:
Yes understand. Just hit this issue today & used the same workaround.